x/emissions/keeper/keeper.go

```diff
@@ -34,21 +34,6 @@ type BLOCK_NUMBER = int64
 type UNIX_TIMESTAMP = uint64
 type REQUEST_ID = string
 
-// Emissions rate Constants
-// TODO make these not constants and figure out how they should
-// be changeable by governance or some algorithm or whatever.
-// Likely should be params within the keeper.
-const EPOCH_LENGTH = 5
```

Comment:
also, think these need to be initialized to something (perhaps just what's here) at genesis, right?

Reply:
they are